### PR TITLE
Done Active Admin's card

### PR DESCRIPTION
**Pull request by gussiciliano**
### Sumary
- Add admin's module with Active Admin

### Trello Card
https://trello.com/c/ZPPMH1GB/10-instalar-y-configurar-active-admin

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class ApplicationController < ActionController::Base
   rescue_from ActionController::ParameterMissing, with: :render_nothing_bad_req
   rescue_from ActiveRecord::RecordNotFound, with: :render_nothing_bad_req
   protect_from_forgery with: :null_session
-  before_action :current_user, :authenticate_request, :set_locale
+  before_action :current_user, :set_locale #, :authenticate_request
```
Comment (Dinuuu) [inline app/controllers/application_controller.rb]: If you comment the authenticate_request method you are not authenticating the request in all the controllers, you should move the authentication logic to another controller that extends from ApplicationController and extend every controller from that one

File: app/controllers/authenticate_request_controller.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    when AuthenticationManager::SUCCESS_CODE
+      200
+    end
+  end
+
+  def render_nothing_bad_req
+    head :bad_request
+  end
+
+end
```
Comment (Dinuuu) [inline app/controllers/authenticate_request_controller.rb]: Enter new line at the of file

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,4 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+AdminUser.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password')
```
Comment (Dinuuu) [inline db/seeds.rb]: Idem

File: spec/models/admin_user_spec.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (Dinuuu) [inline spec/models/admin_user_spec.rb]: If you are not using this file, just delete it

File: app/controllers/authenticate_request_controller.rb (hunk 2 above)
Comment (gussiciliano) [inline app/controllers/authenticate_request_controller.rb]: Oks

File: db/seeds.rb (hunk 3 above)
Comment (gussiciliano) [inline db/seeds.rb]: Oks

File: spec/models/admin_user_spec.rb (hunk 4 above)
Comment (gussiciliano) [inline spec/models/admin_user_spec.rb]: Oks

**Pull request merged**